types/trigger.d.ts

```diff
@@ -130,12 +135,8 @@ export type TriggerSet<Data> = {
   timeline?: TimelineFunc;
   triggers?: NetRegexTrigger<Data>[];
   timelineTriggers?: TimelineTrigger<Data>[];
-  timelineReplace?: {
-    locale: Lang;
-    missingTranslations?: boolean;
-    replaceText?: { [regex: string]: string };
-    replaceSync?: { [regex: string]: string };
-  }[];
+  timelineReplace?: Replacement[];
+  timelineStyles?: Style[];
```

Comment:
I know it's a bit long, but could you call these `TimelineReplacement` and `TimelineStyle`? I think the bare words on their own are ambiguous and could collide with other things.

Reply:
These were pulled in from timeline.ts, they would need to be renamed there, so as long as it's fine to pull that change into this PR, sure.

Reply:
Sure!